dev/com.ibm.websphere.appserver.features/visibility/public/jakartaeeClient-9.1/resources/l10n/io.openliberty.jakartaeeClient-9.1.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2015, 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables support for Jakarta EE 9.1 Application Client.
```

Comment:
for consistency with the other en messages in this PR:

This feature enables support for Jakarta EE 9.1 Application Client.
--->
This feature enables support for the Jakarta EE 9.1 Application Client.

Reply:
Opened https://github.com/OpenLiberty/open-liberty/issues/17563 to track this change (might need making in other versions too)